hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -1311,6 +1311,20 @@
 	String HBM2DDL_FILTER_PROVIDER = "hibernate.hbm2ddl.schema_filter_provider";
 
 	/**
+	 * Used to specify the {@link org.hibernate.tool.schema.spi.SchemaMigrator} to be used to
+	 * perform a schema migration (update) of the database schema.
+	 *
+	 */
+	String HBM2DDL_SCHEMA_MIGRATOR = "hibernate.hbm2ddl.schema_migrator";
+
+	/**
+	 * Used to specify the {@link org.hibernate.tool.schema.spi.SchemaValidator} to be used to
+	 * validate a database schema.
+	 *
+	 */
+	String HBM2DDL_SCHEMA_VALIDATOR = "hibernate.hbm2ddl.schema_validator";
+
+	/**
```

Comment:
@sebersole 
2 reasons, first I have not thought about it :) and second not sure adding the ability to swap the validator and migrator delegates is the best solution, may be as you suggested me a boolean config setting may be better.
Not sure about the config property name any idea?